anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -646,3 +653,46 @@ def transform(self, frames, show_progress=False, normalize=True):
             return normalized_features
         else:
             return features
+
+    def power_spectrum(self, ell_frames, AniSOAP_HYPERS=None):
```

Comment:
Looks generally good -- I would add some docstrings to describe what's the role of this function.